load-tests/load_tests/socketio_locustfile.py

```diff
@@ -0,0 +1,66 @@
+import os
+import time
+from pathlib import Path
+from typing import List
+
+from locust import Locust, TaskSet, between, task
+from rasa_integration_testing.helper import generate_tracker_id_from_scenario_name
+from rasa_integration_testing.interaction import InteractionLoader
+from rasa_integration_testing.scenario import (
+    Scenario,
+    ScenarioFragmentLoader,
+    load_scenarios,
+)
+from socketio import Client
+
+from load_tests.constants import SESSION_ID_KEY
+from load_tests.runner import LocustRunner
+
+ENV_INTEGRATION_TEST_PATH = "INTEGRATION_TEST_PATH"
+ENV_LOAD_TEST_TARGET_URL = "LOAD_TEST_TARGET_URL"
+ENV_MINIMUM_WAIT_TIME = "MINIMUM_WAIT_TIME"
+ENV_MAXIMUM_WAIT_TIME = "MAXIMUM_WAIT_TIME"
+DEFAULT_MINIMUM_WAIT_TIME = 0
+DEFAULT_MAXIMUM_WAIT_TIME = 5
+
+SCENARIOS_GLOB = "*.yml"
+SCENARIOS_FOLDER = "scenarios"
+TEST_CONFIG_FILE = "config.ini"
+
+tests_path = os.environ.get(ENV_INTEGRATION_TEST_PATH, "../integration-tests-en")
```

Comment:
"../integration-tests-en" is now an invalid default

Reply:
Not entirely because it's only relevant for the Docker image. But I do think it's better to have this explicitly set so I removed the default anyways and added the variable in `.env.example`